packages/scipy/meta.yaml

```diff
@@ -111,6 +111,7 @@ requirements:
   host:
     - numpy
     - CLAPACK
+    - boost-cpp
```

Comment:
boost is not a hard dependency of scipy, but it seems like scipy uses it if it is found during build.